src/Fixer/ClassNotation/FinalClassFixer.php

```diff
@@ -44,6 +44,16 @@ class MyApp {}
         );
     }
 
+    /**
+     * {@inheritdoc}
+     *
+     * Must run before ProtectedToPrivateFixer, SelfStaticAccessorFixer.
+     */
+    public function getPriority(): int
+    {
+        return parent::getPriority();
+    }
```

Comment:
my very first thought on seeing such code would be to remove it

Reply:
yes, i know, you have reasons due to that other PR. but still, very counter-intuitive why it's needed at all, when looking on the code itself

Reply:
Is it required here only because of phpDoc with dependent fixers (which is checked by auto-review test)?

Reply:
That parent is `AbstractProxyFixer` that you didn't want to have as an exception.

If you want to remove it let's have a separate PR with proper auto review test since:
```bash
PHP-CS-Fixer(master)$ git grep 'return parent::getPriority();' | wc -l
9
```